src/grisp_tools_util.erl

```diff
@@ -27,11 +27,8 @@ env(Key) ->
     Value.
 
 cdn_path(otp, #{board := Board} = State) ->
-    #{path := Path} = URI = uri_string:parse(env(cdn)),
     File = grisp_tools_util:package_name(State) ++ ".tar.gz",
-    uri_string:recompose(URI#{
-        path => Path ++ string:join(["/platforms", Board, "otp", File], "/")
-    }).
+    env(cdn) ++ string:join(["/platforms", Board, "otp", File], "/").
```

Comment:
`string:join([env(cdn), "platforms", Board, "otp", File], "/")` would be more idiomatic here?